WebView/app/src/main/assets/main.js

```diff
@@ -14,26 +14,18 @@
  * limitations under the License.
  */
 function sendAndroidMessage() {
-	Weather.sendMessage("The weather in WebView City today is: " + document.getElementById("description").innerHTML);
+	Weather.sendMessage(`The weather in ${document.getElementById("title").innerText} today is
+	${document.getElementById("description").innerText} `);
 }
 
 function getData() {
 	fetch("https://gcoleman799.github.io/Asset-Loader/weather.json").then(function(resp) {
 		return resp.json();
+	}).then(function(data) {
+		var form = document.getElementById("location");
+		var currentLocation = form.options[form.selectedIndex].value;
+		document.getElementById("title").innerText = form.options[form.selectedIndex].text;
+        document.getElementById("description").innerText = data[currentLocation].description;
```

Comment:
Please use consistent indentation styles (it looks like your code is using tabs already, so that would be better for this PR).

https://google.github.io/styleguide/jsguide.html#formatting-block-indentation suggests using 2 spaces, so it would be good to send a follow-up PR to reformat the existing lines.

Reply:
Ok